Plugins/GitHub3/GitHubRepo.cs

```diff
@@ -134,9 +118,9 @@ public int CreatePullRequest(string myBranch, string remoteBranch, string title,
             return pullRequest.Number;
         }
 
-        public string CloneProtocol { get; set; } = "SSH";
+        public GitProtocol CloneProtocol { get; set; } = GitProtocol.Ssh;
 
-        public string[] SupportedCloneProtocols { get; set; } = new string[] { "SSH", "HTTPS" };
+        public IReadOnlyList<GitProtocol> SupportedCloneProtocols { get; set; } = new List<GitProtocol> { GitProtocol.Ssh, GitProtocol.Https };
```

Comment:
Do we actually need to create a list here?
Will the following work?
```suggestion
        public IReadOnlyList<GitProtocol> SupportedCloneProtocols { get; set; } = new[] { GitProtocol.Ssh, GitProtocol.Https };
```

Reply:
Surprisingly (because I though I had tested it days ago without success!), yes.